go/chat/server.go

```diff
@@ -1471,6 +1442,25 @@ func (h *Server) GenerateOutboxID(ctx context.Context) (res chat1.OutboxID, err
 	return storage.NewOutboxID()
 }
 
+func (h *Server) runSlashCommands(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID,
+	tlfName string, body chat1.MessageBody) (bool, error) {
+	if !body.IsType(chat1.MessageType_TEXT) {
+		return false, nil
+	}
+	text := body.Text().Body
```

Comment:
short circuit if it doesn't start with `/` so we don't have to try to match everything